code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -243,10 +243,10 @@
 /mob/living/simple_animal/hostile/carp/advanced
 	name = "advanced space carp"
 	desc = "A ferocious, fang-bearing creature that resembles a fish."
-	maxHealth = 40
-	health = 40
+	maxHealth = 60
```

Comment:
This increases (40 to 60 and 15 to 20) are so minor that they probably aren't going to make any real impact.

Reply:
Fair enough, I think 80 and 25 would be better.